AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/AWSS3StoragePluginOptions.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+public struct AWSS3StoragePluginOptions {
+    public let defaultAccessLevel: StorageAccessLevel
```

Comment:
can you mark `deafultAccessLevel` as deprecated?

Reply:
sorry, this options struct should not be added since `defaultAccessLevel` is being deprecated as part of the storage path PR, which leaves this options struct empty. we can introduce the options struct pattern once we do have some client configuration values in the future.

Reply:
Can you remove this change/file then? It still shows the in the latest version

Reply:
I've removed `AWSS3StoragePluginOptions.swift` file, can you check again? might be a delay from Github

Reply:
storage source files <img width="291" alt="image" src="https://github.com/aws-amplify/amplify-swift/assets/1365977/f1422597-6a7c-4488-986d-e52ded6fbae1">